articles/flows/guides/device-auth/includes/troubleshooting.md

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
+
+[Tenant logs](/logs#log-data-event-listing) are created for any interaction that takes place and can be used to troubleshoot issues.
+
+## Error codes
+
+| Code       | Name | Description |
+|------------|------|-------------|
+| `limit_dc` | Too Many Invalid Device Codes	| |
+| `fdeaz`    | Failed device authorization request | |	
+| `fdeac`    | Failed device activation | |	
+| `fede`     | Failed Exchange | Device Code for Access Token |
```

Comment:
There is another one: 

fdecc | User canceled the device confirmation.
-- | --